### PR TITLE
Create the blog detail template

### PR DESCRIPTION
**Pull request by SayyidMuhammad878**

### CODE REVIEW

File: pages/blog/[blogId].js [hunk 1]
```diff
@@ -1,7 +1,94 @@
...
+					<br />
+
+					<p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Repellendus minus quo sed officia quod veritatis tempore molestiae ab accusantium tempora necessitatibus eligendi saepe est, fugiat qui sequi excepturi aspernatur nulla natus rerum quisquam officiis porro! Quas tempora corrupti provident voluptas ut at amet magni quia, totam quam id voluptate, omnis ipsum nesciunt vel similique dicta eum! Ab molestiae modi vero consectetur ratione perspiciatis ipsum, impedit iste at! Nostrum nisi eum illum sed perferendis distinctio quis reprehenderit iusto, expedita animi alias labore, dicta mollitia molestiae hic totam excepturi sequi? Quod, aperiam delectus, quos quasi officiis quas eaque suscipit minima enim quam tempore beatae aut numquam! Architecto reprehenderit rerum hic voluptatem non ullam et deserunt ducimus, praesentium, voluptatibus quod quae eos laborum accusantium veritatis dicta ex, quasi maiores? Unde, necessitatibus debitis quos culpa porro velit, iure aut dolores repudiandae veniam cupiditate. Non repudiandae quidem soluta? Aspernatur quo ab corporis dicta natus facere!
+					</p>
+				</div>
+			</div>
+			<div className={styles.otherBlogsCard}>
```
Comment (sayyedarib) [inline pages/blog/[blogId].js:78]: align other blogs section into center

**Comment by SayyidMuhammad878** [comment]
> it's better to keep 'see other blog section' in center

Done!

**Comment by anmode** [comment]
@SayyidMuhammad878 Please resolve the conflicts in your branch!

**Comment by SayyidMuhammad878** [comment]
> @SayyidMuhammad878 Please resolve the conflicts in your branch!

Done

**Comment by anmode** [comment]
Great work! @SayyidMuhammad878

**Pull request merged**